### ISSUE TITLE
After Pruning, the model size became larger.

### ISSUE THREAD

**Issue by Seokjoongg** [microsoft/nni#3094]
**Environment**:
- NNI version: 1.9
- NNI mode (local|remote|pai): local
- Client OS: Centos 7
- Server OS (for remote mode only):
- Python version: 3.6
- PyTorch/TensorFlow version: Pytorch 1.5.1
- Is conda/virtualenv/venv used?: virtualenv
- Is running in Docker?: No

message:
[INFO] Customize rule count_convNd_mask() <class 'torch.nn.modules.conv.Conv2d'>.
[WARN] Cannot find rule for <class 'nni.compression.torch.compressor.PrunerModuleWrapper'>. Treat it as zero Macs and zero Params.
[INFO] Register count_bn() for <class 'torch.nn.modules.batchnorm.BatchNorm2d'>.
[WARN] Cannot find rule for <class 'torch.nn.modules.container.Sequential'>. Treat it as zero Macs and zero Params.
[WARN] Cannot find rule for <class '__main__.Block'>. Treat it as zero Macs and zero Params.
[INFO] Customize rule count_linear_mask() <class 'torch.nn.modules.linear.Linear'>.
[WARN] Cannot find rule for <class '__main__.MobileNetV2'>. Treat it as zero Macs and zero Params.

**What issue meet, what's expected?**:
The model was compressed using the pruning function built into nni. (FPGM, L1, Slim etc.) However, the size of the model file has always doubled no matter what compression rate is assumed. When I used nni's flops counter, I found that flops decreased but the number of parameters did not decrease. Why is this? The model was saved in pth format.

**Comment by liuzhe-lz** [comment]
By default NNI stores a mask tensor alongside the weight, which doubles the parameter number. This is in favor of debugging and pruning algorithm developing.
When you are satisfied with theoretical accuracy and flops count, you can use [model speedup](https://github.com/microsoft/nni/blob/master/docs/en_US/Compression/ModelSpeedup.md) module to  accelerate your model.

**Comment by Seokjoongg** [comment]
without model speedup module, we can't compress our model?

**Comment by kvartet** [comment]
hello @Seokjoongg, can you follow [this tutorial](https://nni.readthedocs.io/en/latest/Compression/Overview.html#compression-pipeline) and try again? Thank you!

**Comment by scarlett2018** [comment]
as the tutorial has been provided, and there are no responds so far, closing the issue. feel free to reopen if it is still an issue.
